lib/rubocop/cop/rspec/void_expect.rb

```diff
@@ -51,7 +51,8 @@ def void?(expect)
           parent = expect.parent
           return true unless parent
           return true if parent.begin_type?
-          return true if parent.block_type? && parent.body == expect
+
+          true if parent.block_type? && parent.body == expect
```

Comment:
This could be simplified to
```rb
parent.block_type? && parent.body == expect
```

Reply:
Maybe, would it be good to have a cop to offense this?

Reply:
We can add support https://docs.rubocop.org/rubocop/cops_style.html#styleredundantconditional